### ISSUE TITLE
Cannot Set Password (the terminal does not acknowledge keyboard input) C100 (asus flip)

### ISSUE THREAD

**Issue by vodot** [dnschneid/crouton#2507]
```
Please paste the output of the following command here: sudo edit-chroot -all
NA- can't run sude edit-chroot -all

#### Please describe your issue:
cannot set password (terminal refuses keyboard input for any 'password' prompts)

#### If known, describe the steps to reproduce the issue:

Couldn't set it during install (but could hit enter and leave it blank).
Gave it no response enough times for it to leave it unset
Can't set it now using passwd [user] (still refuses keyboard input- and hitting return to leave it blank returns "Authentication token manipulation error"

**Comment by vodot** [comment]
please close; resolved by entering the chroot as root and using `passwd` from there.

**Comment by DennisLfromGA** [comment]
No problem, glad you figured it out and thanx for coming back and closing the issue.

**Comment by vodot** [comment]
Just to clarify and keyword this better for other users who are as new to this as I am: 

My issue was that during install I **did not see anything happening when I was attempting to enter a password.**  I did not realize this was the default behavior of password prompts in the terminal.  I left the password blank (un-set) by repeatedly pressing enter with nothing entered into the password prompt. A message appeared reminding me that I could reset the password afterward using `passwd [user]`, but was not able to do so.

I resolved this issue by a) understanding that the password prompt was still recording my keystrokes even though it was not showing me anything, and b) **logging into the chroot as root** with `sudo enter-chroot -u root` and then running `passwd [user]` from there. 

Essentially, this issue duplicates [issue #705](https://github.com/dnschneid/crouton/issues/705)
